rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -430,7 +438,9 @@ def test_export_snapshot(rotkehlchen_api_server, tmpdir_factory):
     with db.user_write() as cursor:
         _populate_db_with_balances(cursor, db, ts)
         _populate_db_with_location_data(cursor, db, ts)
-        db.set_settings(cursor, ModifiableDBSettings(main_currency=A_EUR))
+        db.set_settings(cursor, ModifiableDBSettings(
+            main_currency=A_EUR.resolve_to_asset_with_oracles()),
+            )
```

Comment:
This should be aligned with the line where it opens.

```suggestion
        )
```